copy/copy.go

```diff
@@ -668,10 +663,15 @@ func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, ba
 		}
 		if reused {
 			logrus.Debugf("Skipping blob %s (already present):", srcInfo.Digest)
+			bar := ic.c.createProgressBar(pool, srcInfo, "blob", "completed: already exists")
+			bar.SetTotal(0, true)
```

Comment:
`bar.SetTotal(srcInfo.Size, True)`, please, the behavior of `0` is undocumented.

Reply:
… never mind, we have literally copied 0 bytes, so `0` is semantically correct.

(Although it does not actually set the total to 0… but we are removing the bar and the counters on completion anyway, so no-one can ever notice.)

Reply:
Good catch!